### DISCUSSION TITLE
Adding rule to switch out `@octokit/webhooks-types` with `@octokit/openapi-webhooks-types` on upgrade of `@octokit/webhooks` to v13

### DISCUSSION THREAD

**Discussion by wolfy1339** [renovatebot/renovate#28253] [Suggest an Idea]
### Tell us more.

Would it be possible to add a rule to renovate to switch out some packages when upgrading `@octokit/webhooks` to v13?

If the `@octokit/webhooks-types` package is present, switch it out for the `@octokit/openapi-webhooks-types` package instead

The reason for the switch is because we are now using GitHub's OpenAPI spec instead of community generated specs

**Comment by rarkins** [comment]
Yes, the capability exists. PRs welcome to add it to default replacements
